integrations/great_expectations/fraud_batch_data_validation.ipynb

```diff
@@ -0,0 +1,3478 @@
+        "\n",
+        "Feature groups can also be used to define a namespace for features. For instance, in a real-life setting we would likely want to experiment with different window lengths. In that case, we can create feature groups with identical schema for each window length. \n",
+        "\n",
+        "Before we can create a feature group we need to connect to our feature store."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 30,
+      "id": "0a9cde37",
+      "metadata": {
+        "id": "0a9cde37",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "668c684a-8b90-446a-acb3-6c2c86865808"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Connected. Call `.close()` to terminate connection gracefully.\n",
+            "Connection closed.\n",
+            "Copy your Api Key (first register/login): https://c.app.hopsworks.ai/account/api/generated\n",
+            "\n",
+            "Paste it here: I1GJfbJAfL2WQNrb.RA1oJvSmLXCLFX1G669lczYnaJ08oSotWNsi7lAgpoWYOQPgfZP6zLeutkdCQRk0\n",
+            "Connected. Call `.close()` to terminate connection gracefully.\n",
+            "\n",
+            "Logged in to project, explore it here https://c.app.hopsworks.ai:443/p/155\n",
+            "Connected. Call `.close()` to terminate connection gracefully.\n"
+          ]
+        },
+        {
+          "output_type": "stream",
+          "name": "stderr",
+          "text": [
+            "DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working\n",
+            "DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working\n"
+          ]
+        }
+      ],
+      "source": [
+        "import hopsworks\n",
+        "\n",
+        "project = hopsworks.login()\n",
+        "\n",
+        "fs = project.get_feature_store()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "eef128e1",
+      "metadata": {
+        "id": "eef128e1"
+      },
+      "source": [
+        "To create a feature group we need to give it a name and specify a primary key. It is also good to provide a description of the contents of the feature group and a version number, if it is not defined it will automatically be incremented to `1`."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 40,
+      "id": "fd67e4c8",
+      "metadata": {
+        "id": "fd67e4c8"
+      },
+      "outputs": [],
+      "source": [
+        "trans_fg = fs.get_or_create_feature_group(\n",
+        "    name=\"transactions_fraud_batch_fg\",\n",
```

Comment:
To avoid collisions with other notebooks, can we add "_ge_" or something somewhere in the name.

Reply:
Good point, will do